apps/allocations/app/components/Panel/NewBudget.js

```diff
@@ -91,7 +99,8 @@ class NewBudget extends React.Component {
       <Form
         onSubmit={this.createBudget}
         submitText={buttonText}
-        disabled={nameError || amountError || amountOverFunds}
+        disabled={name === '' || amount === '' || nameError || amountError
```

Comment:
This change doesn't feel directly related to the overarching PR, the line length feels handled weird, and now there's two conditions that will disable the submit without any sort of user messaging. Still probably better to have this check in here and I'm not sure that building this out falls under the scope of this PR.